src/app/garfile/arrival/index.njk

```diff
@@ -132,12 +132,12 @@
           {{ m.error_message('arrivalPort', errors, __) }}
           <input class="govuk-input govuk-input--width-4" maxlength="4" id="arrivalPort" name="arrivalPort"
             autocomplete="off" type="text" value="{{cookie.getGarArrivalVoyage().arrivalPort | upper }}"
-            aria-describedby="arrivalPort-hint arrivalPort-error"/>
+            aria-describedby="arrivalPort-hint{{ ' arrivalPort-error' if errors | containsError('arrivalPort') }}"/>
         </div>
         {# End arrival port #}
 
         <div class="govuk-form-group">
-          <label class="govuk-heading-m">{{__('field_arrival_coordinates')}}</label>
```

Comment:
WAVE mentioned a label without a "for" is enough to be an error, so just changed to a span here.